### PR TITLE
E

### PR DESCRIPTION
**Pull request by hYdos**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,8 +9,7 @@ function createWindow () {
     }
   })
 
-  win.loadFile('site/index.html')
-  win.webContents.openDevTools()
+  win.loadFile('sites/waffle/index.html')
```
Comment (hYdos) [inline index.js]: please fix thank

File: sites/waffle/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (hYdos) [inline sites/waffle/index.html]: dont delete site/index.html thank

**Review by hYdos — COMMENTED** [review]
Please fix these

**Comment by Arc-blroth** [comment]
fixed™️

File: index.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ function createWindow () {
       nodeIntegration: true
     }
   })
-
+
```
Comment (hYdos) [inline index.js:11]: why

Comment (Arc-blroth) [reply index.js:11]: yes

**Pull request merged**